react-ui/src/pages/contest/Contest.jsx

```diff
@@ -314,446 +146,70 @@ function Contest() {
-                  mutate();
-                }}
-                submitting={isValidating}
-                variant="contained"
-              >
-                Refresh
-              </SpinnerButton>
-            </>
-          )}
+          {votingWindowOpen === false && <ContestUnderReview {...{ isValidating, mutate }} />}
           {isContestMode && subtext && (
             <Box marginBottom={3}>
               <Typography component="div" variant="subtitle1">
                 <HtmlWrapper html={subtext} />
               </Typography>
             </Box>
           )}
-          {categories?.length > 0 && (
-            <div className={classes.categories}>
-              <Typography id={FILTER_CATEGORIES_LABEL_ID} variant="caption">
-                Filter categories:
-              </Typography>
-              <Select
-                input={<Input />}
-                labelId={FILTER_CATEGORIES_LABEL_ID}
-                MenuProps={CATEGORY_MENU_PROPS}
-                multiple
-                onChange={handleCategoryChange}
-                renderValue={(selected) => (
-                  <Box display="flex" flexWrap="wrap">
-                    {selected.map((value) => (
-                      <Chip
-                        className={clsx(
-                          classes.categoryChip,
-                          categoryLabelClasses[
-                          // eslint-disable-next-line indent
-                          `label${categories.indexOf(value) % LABEL_COLORS.length}`
-                          ],
-                        )}
-                        key={value}
-                        label={value}
-                      />
-                    ))}
-                  </Box>
-                )}
-                value={selectedCategories}
-              >
-                {categories.map((category, index) => (
-                  <MenuItem
-                    className={clsx({
-                      [classes.selectedCategory]: selectedCategories.includes(category),
-                    })}
-                    key={category}
-                    value={category}
-                  >
-                    <div
-                      className={clsx(
-                        classes.categoryLabel,
-                        categoryLabelClasses[`label${index % LABEL_COLORS.length}`],
-                      )}
-                    >
-                      {category}
-                    </div>
-                  </MenuItem>
-                ))}
-              </Select>
-              <Button
-                color="primary"
-                disabled={!selectedCategories.length}
-                size="small"
-                onClick={resetSelectedCategories}
-              >
-                Reset
-              </Button>
-            </div>
-          )}
+          <ContestCategorySelector {...{ categories, selectedCategories, setSelectedCategories }} />
           {winners && winners.length > 0 && (
```

Comment:
Ya I did not focus on the winners because it seemed legacy-based, and figured at some point we would adjust those contests from the database side and eliminate winners altogether. But I updated `ContestWinners` to match.